setup.py

```diff
@@ -1,24 +1,31 @@
-import json
-import setuptools
+"""
+XBlock for linking to a Qualtrics survey
+"""
+from setuptools import setup
+import os
 
 
-package_json_file = open('package.json', 'r')
-package_json = json.load(package_json_file)
+os.environ['DJANGO_SETTINGS_MODULE'] = 'settings.base'
+description = __doc__
 
-setuptools.setup(
-    name=package_json.get('name', 'xblock-test'),
-    version=package_json.get('version', '0.1.1'),
-    description=package_json.get('description'),
-    long_description=package_json.get('description'),
-    author=package_json.get('author', {}).get('name'),
-    author_email=package_json.get('author', {}).get('email'),
-    url=package_json.get('homepage'),
+setup(
+    name='xblock_qualtrics_survey',
```

Comment:
It didn't make sense to keep `package.json` as the canonical form of truth here.
NPM is a second class citizen in an XBlock.